src/shared/copy/notifications/categories/accounts-users-orgs.ts

```diff
@@ -130,3 +120,14 @@ export const removeUserSuccessful = (): Notification => ({
   ...defaultSuccessNotification,
   message: `User Removed`,
 })
+
+export const userProfileSaveSuccess = (): Notification => ({
+  ...defaultSuccessNotification,
+  message: 'Saved changes to your profile.',
+})
+
+export const userProfileSaveError = (): Notification => ({
+  ...defaultErrorNotification,
+  message:
+    'Failed to save all changes to your profile. Please try again later.',
```

Comment:
We might consider dropping `later`. That always makes me think I should come back in a couple of hours or days.